components/vocabularies/rest/src/main/java/org/phenotips/vocabularies/rest/CategoriesResource.java

```diff
@@ -0,0 +1,51 @@
+/*
+ * See the NOTICE file distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see http://www.gnu.org/licenses/
+ */
+package org.phenotips.vocabularies.rest;
+
+import org.phenotips.rest.ParentResource;
+import org.phenotips.rest.Relation;
+import org.phenotips.vocabularies.rest.model.Categories;
+
+import org.xwiki.stability.Unstable;
+
+import javax.ws.rs.GET;
+import javax.ws.rs.Path;
+import javax.ws.rs.Produces;
+import javax.ws.rs.core.MediaType;
+
+/**
+ * Resource for working with vocabulary categories, listing all the available categories.
+ *
+ * @version $Id $
```

Comment:
`$Id $` is not valid, git won't recognize it. Use `$Id` (no space). Is this autogenerated, in which case your IDE settings would need to be adjusted?